Added rpm parsing tool. Modified Package Object for optional package state (#121)
netmeld/netmeld: datastore/importers/nmdb-import-rpm/README.md

DESCRIPTION
===========

> Parse and import the output of the `rpm -qa --queryformat="%-50{NAME}%10{VERSION}%-%-20{RELEASE}%-20{ARCH}%{SUMMARY}\n"` command on modern Linux
> systems into the Netmeld framework for later analysis. Import's package's
> based on device-id.

Reviewer: ```suggestion
Parse and import the output of the
`rpm -qa --queryformat="%-50{NAME}%10{VERSION}%-%-20{RELEASE}%-20{ARCH}%{SUMMARY}\n"`
command on modern Linux systems into Netmeld for later analysis.
The imported package list is associated with the passed `--device-id` value.
```
